internal/utils/jobhelper.go

```diff
@@ -127,3 +129,31 @@ func (jh *jobHelper) GetJobStatus(job *batchv1.Job) (Status, bool, error) {
 		return StatusFailed, false, fmt.Errorf("unknown status: %v", job.Status)
 	}
 }
+
+func (jh *jobHelper) getJobs(ctx context.Context, namespace string, labels map[string]string) ([]batchv1.Job, error) {
+	jobList := batchv1.JobList{}
+	opts := []client.ListOption{
+		client.MatchingLabels(labels),
+		client.InNamespace(namespace),
+	}
+	if err := jh.client.List(ctx, &jobList, opts...); err != nil {
+		return nil, fmt.Errorf("could not list jobs: %v", err)
+	}
+
+	return jobList.Items, nil
+}
+
+func moduleKernelLabels(moduleName, targetKernel, jobType string) map[string]string {
+	return map[string]string{
+		constants.ModuleNameLabel:    moduleName,
+		constants.TargetKernelTarget: targetKernel,
+		constants.JobType:            jobType,
+	}
```

Comment:
```suggestion
	m := moduleLabels(moduleName, jobType)
	m[constants.TargetKernelTarget] = targetKernel
	
	return m
```

Reply:
unless you insist, i prefer to keep it as is: i think it is slightly more readable, since you see immediately all the labels, without need to check what moduleLabels function does

Reply:
Aren't you afraid that at some point we update one but forget to update the other?